src/mgr/ClusterState.h

```diff
@@ -27,6 +28,25 @@ class MMgrDigest;
 class MMonMgrReport;
 class MPGStats;
 
+class C_ClientInMgrMapCond : public C_SaferCond {
+  std::string_view name;
+  const entity_addrvec_t& client_addr;
```

Comment:
```suggestion
  std::string name;
  entity_addrvec_t client_addr;
```

It's too dangerous to have the member lifetime managed outside the class. It's not worth whatever small performance benefit it provides.